### PR TITLE
add header

### PR DESCRIPTION
**Pull request by vuphuctho**
Added basic dependencies to projects
Added static header for index.html

### CODE REVIEW

**Comment by trungnn** [comment]
Commit message should be capitalized and be more descriptive.

[Commit message guide](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html)

**Comment by trungnn** [comment]
What's the use of `bundle.js`? If that's a generated file from some build scripts/tools then you don't need to commit that. Also, if that's the case then you can safely put the file to `.gitignore` to avoid pushing it everytime you make any changes.

File: js/components/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+var React = require("react");
```
Comment (trungnn) [inline js/components/Navbar.js:1]: Can use `const` instead of `var` here, and for other parts as well?
Because ES6 😛

Comment (vuphuctho) [reply js/components/Navbar.js:1]: Will follow ES6 standard in the next commits

**Pull request merged**